day03/ex00/ex00.md

```diff
@@ -0,0 +1,143 @@
+# Exercise 00 - AWS setup.
+
+|                         |                    |
+| -----------------------:| ------------------ |
+|   Turnin directory :    |  ex00              |
+|   Files to turn in :    |                    |
+|   Forbidden function :  |  None              |
+|   Remarks :             |  n/a               |
+
+## What is the Cloud ?
+
+Cloud computing is the on-demand delivery of IT resources and applications via the Internet with pay-as-you-go pricing. In reality, a cloud server is in a data center all around the world.
+
+Whether you run applications that share photos to millions of mobile users or deliver services that support the critical operations of your business, the cloud provides rapid access to flexible and low-cost IT resources. With cloud computing, you don’t need to make large up-front investments in hardware and spend a lot of time managing that hardware. Instead, you can provision exactly the right type and size of computing resources you need to power your newest bright idea or operate your IT department. With cloud computing, you can access as many resources as you need, almost instantly, and only pay for what you use.
+
+In its simplest form, cloud computing provides an easy way to access servers, storage, databases, and a broad set of application services over the Internet. Cloud computing providers such as AWS own and maintain the network-connected hardware required for these application services, while you provision and use what you need for your workloads.
+
+As seen previously, Cloud computing has some real benefits :
+
+- **variable expense** : we don't need to invest in huge data centers you may not use. You pay for how much you consume !
+- **available in minutes** : new IT resources can be accessed within minutes.
+- **economies of scale** : because of the number of users. Cloud providers can achieve higher economies of scale translating in lower prices.
```

Comment:
```suggestion
- **economies of scale**: because of the number of users, Cloud providers can achieve higher economies of scale translating at lower prices.
```